lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -143,6 +144,8 @@ void HostManager::removeNetwork(llvm::StringRef networkName) {
     provisioner_->removeFunction(node->name);
   }
   networks_.erase(networkIterator);
+
+  return llvm::Error::success();
```

Comment:
Not sure whether evictNetwork error of each device to be considered for returning success status at the end.

Reply:
Yes I think it should. You can use OneErrOnly to return the first Error to occur